client/layout/index.jsx

```diff
@@ -420,6 +429,8 @@ export default withCurrentRoute(
 			isOffline: isOffline( state ),
 			currentLayoutFocus: getCurrentLayoutFocus( state ),
 			colorScheme,
+			needsColorScheme: ! shouldShowGlobalSidebar,
```

Comment:
This condition is wrong. Will investigate later...

Reply:
Basically got bitten by the convoluted logic; see paYKcK-5Qk-p2 😅

Reply:
Dang. :(